org.d3cod3.Mosaic.yml

```diff
@@ -0,0 +1,438 @@
+app-id: org.d3cod3.Mosaic
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: mosaic
+# rename-icon: accessories-dictionary
+finish-args:
```

Comment:
You miss `--device=dri` but it fails shortly after anyway:

```
/usr/include/c++/10.2.0/bits/stl_vector.h:1045: std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::operator[](std::vector<_Tp, _Alloc>::size_type) [with _Tp = ofSoundDevice; _Alloc = std::allocator<ofSoundDevice>; std::vector<_Tp, _Alloc>::reference = ofSoundDevice&; std::vector<_Tp, _Alloc>::size_type = long unsigned int]: Assertion '__builtin_expect(__n < this->size(), true)' failed.
```

My uneducated guess would be one of the default GCC flags causes it to (correctly!) segfault on faulty code, likely one of `-Wp,-D_GLIBCXX_ASSERTIONS -fexceptions`.